examples/lb3-application/lb3app/server/config.json

```diff
@@ -1,11 +1,10 @@
 {
-  "restApiRoot": "/api",
-  "host": "0.0.0.0",
-  "port": 3000,
   "remoting": {
+    "errorHandler": false,
```

Comment:
On the second thought, I am not sure if this is necessary, considering that we are setting `rest.handleErrors` to `false` few lines below. Maybe my original suggestion was based on docs for LB2.